docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<blockquote>T(+) looks noisy, 𝐓(+) has to include contains  at  least  two  instances</blockquote><p> </p><p>Please rewrite it as follows: <code>T(+) looks noisy. It contains at least two instances that are of our interest.</code> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>FIx</p>